features/sharding/core/src/test/java/org/apache/shardingsphere/sharding/route/engine/type/standard/ShardingStandardRoutingEngineTest.java

```diff
@@ -45,7 +44,6 @@ public final class ShardingStandardRoutingEngineTest extends AbstractRoutingEngi
     
     @After
     public void tearDown() {
```

Comment:
Can this method `tearDown()` be removed?

Reply:
ok,and i removed HintManager & HintManagerHolder,is it OK？

Reply:
> Can this method `tearDown()` be removed?
ok,and i removed HintManager & HintManagerHolder,is it OK？

Reply:
> Can this method `tearDown()` be removed?

ok,and i removed HintManager & HintManagerHolder,is it OK？

Reply:
@anmui Yes, they should be removed.